tests/test_georaster.py

```diff
@@ -1028,7 +1039,7 @@ def test_numpy_functions(dtype: str) -> None:
     transform = rio.transform.from_origin(0, 5, 1, 1)
 
     # Create a raster from the array
-    raster = gu.Raster.from_array(array, transform=transform, crs=4326)
+    raster = gu.Raster.from_array(np.ma.masked_array(array), transform=transform, crs=4326)
```

Comment:
Why is np.ma.masked_array needed each time? If so, why isn't it enforced in the `from_array` method directly? It's a bit annoying to have to write it each time...

Reply:
We can enforce masked arrays! But I feel like that has to be a unanimous decision.

Reply:
And in the meantime, would the tests not pass without the conversion to a masked array?

Reply:
I added these because you call the mask as if it most definitely exists, which it (right now) doesn't necessarily. For example (a bit further down):

```python
if np.sum(r3.data.mask) == 0:
```

Reply:
Stupid past me ! Fortunately I'm getting brighter everyday 😆

Reply:
These changes are reverted in 3e84043 as `Raster.data` is now always a masked_array